src/renderer/views/Playlist/Playlist.js

```diff
@@ -135,6 +141,10 @@ export default Vue.extend({
           channelId: result.authorId,
           infoSource: 'invidious'
         }
+        const channelThumbnailUrl = result.authorThumbnails[2].url
+        const channelName = this.infoData.channelName
+        const channelId = this.infoData.channelId
+        this.updateSubscriptionDetails({ channelThumbnailUrl, channelName, channelId })
```

Comment:
Please inline these variables as you did in https://github.com/FreeTubeApp/FreeTube/pull/1783/commits/ba537cc1b8572ce20467634bd5dcf7baf0b7eb67.